<launch>

  <arg name="robot" default="iwo_wjx"/>

  <group ns="$(arg robot)">
    <node pkg="nodelet" type="nodelet" name="eskf_iwo"
      args="standalone eskf_iwo/EskfIwoNodelet"
      output="screen">

    </node>
  </group>

</launch>
